<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>16</width>
     <height>501</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tm1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Использование редактора тайлов</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic;&quot;&gt;Руководство редактора&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;Руководство редактора&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm1_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Очки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm1_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Размер карты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm1_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Дистанция</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Свойства атаки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm2_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Delay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm2_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Типы атаки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm3_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Do</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm3_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>Dpsa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tm3_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 170, 255);
border: none;</string>
   </property>
   <property name="text">
    <string>ida</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
